manifests/storage-class.yaml

```diff
@@ -5,5 +5,4 @@ metadata:
   name: default
 provisioner: kubernetes.io/aws-ebs
 parameters:
-  type: gp2
-  encrypted: "true"
+  persistent_disk_type: kubernetes_storage_class
```

Comment:
This is a kubernetes manifest, not a bosh manifest, so `persistent_disk_type` doesn't have any meaning here. This change should be reverted.